### PR TITLE
[fr] Fix broken link on Creating a cluster with kubeadm pag

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/37751/files
/assign @remyleone

### CODE REVIEW

File: content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md [hunk 1]
```diff
@@ -676,7 +676,7 @@ si le master est irrécupérable, votre cluster peut perdre ses données et peut
 partir de zéro. L'ajout du support HA (plusieurs serveurs etcd, plusieurs API servers, etc.)
 à kubeadm est encore en cours de developpement.
 
-   Contournement: régulièrement [sauvegarder etcd](https://coreos.com/etcd/docs/latest/admin_guide.html).
+   Contournement: régulièrement [sauvegarder etcd](https://etcd.io/docs/v3.5/op-guide/recovery/).
```
Comment (perriea) [inline content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:679]:
```suggestion
   Contournement: régulièrement [sauvegarder etcd](https://etcd.io/docs/latest/op-guide/recovery/).
```

Comment (perriea) [reply content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:679]: Please don't choose a specific version

**Review by perriea — CHANGES_REQUESTED** [review]

File: content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md (hunk 1 above)
Comment (Arhell) [reply content/fr/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md:679]: I don't mind, but I need to change it in the English version, I'm just doing a little synchronization

**Pull request merged**